getting-started/installation-windows-msys2.rst

```diff
@@ -0,0 +1,290 @@
+Launch the "MSYS MinGW 64-bit" option to use the MinGW 64bit toolchain, producing 64bit binaries that run on 64bit Windows systems. The "MSYS MinGW 32-bit" option will use the MinGW 32bit toolchain, producing 32bit binaries that will run on 32bit and 64bit Windows systems.
+
+Again: you have a single installation of MSYS2, these different shells are just setups to use different compilers. Installation and update of your MSYS2 system only has to be done once - you can use any of the shell options for that.
+
+Update MSYS2 with following command::
+
+    $ pacman -Syu
+  
+After this finishes (let it close the bash shell), open bash again and run the same command again to finish the updates. The same procedure is used for regular updates of the MSYS2 installation. An up-to-date system shows::
+
+    $ pacman -Syu
+    :: Synchronizing package databases...
+     mingw32 is up to date
+     mingw64 is up to date
+     msys is up to date
+    :: Starting core system upgrade...
+     there is nothing to do
+    :: Starting full system upgrade...
+     there is nothing to do
+
+Install the necessary tools (perl is already part of the base system)::
+
+    $ pacman -S tar make
+
+Packages with such "simple" names are part of the MSYS2 environment and work for all compilers/toolchains that you may install on top on MSYS2.
+
+Install compiler toolchains
+---------------------------
+Packages that are part of a MinGW toolchain start with the prefix "mingw-w64-x86_64-" for the 64bit toolchain or "mingw-w64-i686-" for the 32bit toolchain.
+(The "w64" part identifies the host system will be different when using a 32bit MSYS2 environment, e.g. on a 32bit Windows host.)
+
+Install the MinGW 32bit and/or MinGW 64bit toolchains::
+
+    $ pacman -S mingw-w64-x86_64-toolchain
+    $ pacman -S mingw-w64-i686-toolchain
+    
+Each complete toolchain needs about 900MB of disk space.
+If you want to cut down the needed disk space (to about 50%), instead of hitting return when asked which packages of the group to install, you can select the minimal set of packages required for compiling EPICS Base: ``binutils``, ``gcc`` and ``gcc-libs``.
+
+If you are not sure, check your set of tools is complete and everything is installed properly::
+
+    $ pacman -Q
+    ...
+    make 4.3-1
+    perl 5.32.0-2
+    mingw-w64-x86_64-...
+    mingw-w64-i686-...
+    ...
+
+Update your installation regularly
+----------------------------------
+As mentioned above, you can update your complete installation (including all tools and compiler toolchains) at any time using::
+
+    $ pacman -Syu
+
+You should do this in regular intervals.
+
+Download and build EPICS Base
+-----------------------------
+
+::
+
+    $ cd $HOME
+    $ wget https://epics-controls.org/download/base/base-7.0.4.1.tar.gz
+    $ tar -xvf base-7.0.4.1.tar.gz
+    $ cd base-R7.0.4.1
```

Comment:
Can we rename EPICS base folder to  "base" ? So, that when new EPICS version is installed, one don't have to set Environment Variables again and restart the system.

In that case,
    $ cd $HOME
    $ wget https://epics-controls.org/download/base/base-7.0.4.1.tar.gz
    $ tar -xvf base-7.0.4.1.tar.gz
    $ mv base-R7.0.4.1 base
    $ cd base
    $ export EPICS_HOST_ARCH=windows-x64-mingw
    $ make

Reply:
You never have to restart the system after setting environment variables.

It is strongly suggested to keep base and support modules in versioned directories. This is how the whole build system is supposed to work: downstream modules switch between versions through their `configure/RELEASE` file. Otherwise, you will never be able to cleanly switch between dependency versions and rebuild modules, and for any given machine or module you will have a hard time finding out which version of base is installed and which version other modules were built against.

Examples should give an example (!) and not establish wrong procedures.